pkg/apis/workflow/v1alpha1/workflow_types.go

```diff
@@ -22,6 +22,8 @@ import (
 	"k8s.io/apimachinery/pkg/util/intstr"
 	"k8s.io/apimachinery/pkg/util/wait"
 
+	"github.com/sirupsen/logrus"
```

Comment:
logrus is always imported as log elsewhere in the codebase
`log "github.com/sirupsen/logrus"`